local/bin/x-compare-versions.py

```
@@ -46,9 +46,25 @@ def test():
+
+    # mixed major/minor version comparisons
+    assert vercmp("2 >= 1.5")
+    assert not vercmp("1 < 1.0")
+
+    # invalid operator should raise an error
+    try:
+        vercmp("1.0 <> 2.0")
+    except KeyError:
+        pass
+    else:
+        assert False, "invalid operator did not raise"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)